src/components/MembershipRequirements/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {View} from 'react-native';
+import {FormattedMessage} from 'react-intl';
+import AppText from '@/components/AppText2';
+import {useTheme} from 'emotion-theming';
+import Requirements from '@/components/Requirements';
+
+import {requirement, container, requirementSection} from './style';
+
+const MembershipRequirements = ({...props}) => {
```

Comment:
If you de-structure nothing, you can just do this:
```suggestion
const MembershipRequirements = props => {
```

Reply:
Also if this component will not be re-used, put under the specific component